stestr/config_file.py

```diff
@@ -62,8 +62,10 @@ def get_run_command(self, options, test_ids=None, regexes=None):
             top_dir = options.top_dir
         elif self.parser.has_option('DEFAULT', 'top_dir'):
             top_dir = self.parser.get('DEFAULT', 'top_dir')
-        command = "${PYTHON:-python} -m subunit.run discover -t" \
-                  " %s %s $LISTOPT $IDOPTION" % (top_dir, test_path)
+
+        python = 'python' if sys.platform == 'win32' else '${PYTHON:-python}'
```

Comment:
So I get that there isn't env variable replacement on windows, but is the python binary always called 'python'?

I guess it doesn't really matter though